admin/app/views/layouts/spree/page_builder.html.erb

```diff
@@ -10,7 +10,6 @@
         data-controller="page-builder"
         data-page-builder-preview-url-value="<%= page_preview_url %>"
   >
-    <%= render 'spree/admin/shared/alerts' %>
```

Comment:
We are already rendering this partial here https://github.com/spree/spree/pull/12572/files#diff-a6951a38ae3a566b66404e47926c27f17fd2e799ebb57a2b1cb47e095f14dd52L61